conseil-common/src/main/scala/tech/cryptonomic/conseil/common/bitcoin/Tables.scala

```diff
@@ -0,0 +1,210 @@
+
+  /** Entity class storing rows of table Inputs
+   *  @param txid Database column txid SqlType(text)
+   *  @param vOut Database column v_out SqlType(int4), Default(None)
+   *  @param scriptSigAsm Database column script_sig_asm SqlType(text), Default(None)
+   *  @param scriptSigHex Database column script_sig_hex SqlType(text), Default(None)
+   *  @param sequence Database column sequence SqlType(int8)
+   *  @param coinbase Database column coinbase SqlType(text), Default(None)
+   *  @param txInWitness Database column tx_in_witness SqlType(_text), Default(None) */
+  case class InputsRow(txid: String, vOut: Option[Int] = None, scriptSigAsm: Option[String] = None, scriptSigHex: Option[String] = None, sequence: Long, coinbase: Option[String] = None, txInWitness: Option[List[String]] = None)
+  /** GetResult implicit for fetching InputsRow objects using plain SQL queries */
+  implicit def GetResultInputsRow(implicit e0: GR[String], e1: GR[Option[Int]], e2: GR[Option[String]], e3: GR[Long], e4: GR[Option[List[String]]]): GR[InputsRow] = GR{
+    prs => import prs._
+    InputsRow.tupled((<<[String], <<?[Int], <<?[String], <<?[String], <<[Long], <<?[String], <<?[List[String]]))
+  }
+  /** Table description of table inputs. Objects of this class serve as prototypes for rows in queries. */
+  class Inputs(_tableTag: Tag) extends profile.api.Table[InputsRow](_tableTag, Some("bitcoin"), "inputs") {
+    def * = (txid, vOut, scriptSigAsm, scriptSigHex, sequence, coinbase, txInWitness) <> (InputsRow.tupled, InputsRow.unapply)
+    /** Maps whole row to an option. Useful for outer joins. */
+    def ? = ((Rep.Some(txid), vOut, scriptSigAsm, scriptSigHex, Rep.Some(sequence), coinbase, txInWitness)).shaped.<>({r=>import r._; _1.map(_=> InputsRow.tupled((_1.get, _2, _3, _4, _5.get, _6, _7)))}, (_:Any) =>  throw new Exception("Inserting into ? projection not supported."))
+
+    /** Database column txid SqlType(text) */
+    val txid: Rep[String] = column[String]("txid")
+    /** Database column v_out SqlType(int4), Default(None) */
+    val vOut: Rep[Option[Int]] = column[Option[Int]]("v_out", O.Default(None))
+    /** Database column script_sig_asm SqlType(text), Default(None) */
+    val scriptSigAsm: Rep[Option[String]] = column[Option[String]]("script_sig_asm", O.Default(None))
+    /** Database column script_sig_hex SqlType(text), Default(None) */
+    val scriptSigHex: Rep[Option[String]] = column[Option[String]]("script_sig_hex", O.Default(None))
+    /** Database column sequence SqlType(int8) */
+    val sequence: Rep[Long] = column[Long]("sequence")
+    /** Database column coinbase SqlType(text), Default(None) */
+    val coinbase: Rep[Option[String]] = column[Option[String]]("coinbase", O.Default(None))
+    /** Database column tx_in_witness SqlType(_text), Default(None) */
+    val txInWitness: Rep[Option[List[String]]] = column[Option[List[String]]]("tx_in_witness", O.Default(None))
```

Comment:
I believe all of the changes, are because of that, but couldn't we think about something less complex? Maybe we could have that done at case-class level. Meaning, put into DB as text, and split it by some predefined delimiter in case class?

Reply:
I'll do it as you said